tests/integration/pilot/common/routing.go

```diff
@@ -953,6 +953,90 @@ spec:
 			},
 			workloadAgnostic: true,
 		})
+
+		// access is denied when the request header `end-user` is not jason.
+		t.RunTraffic(TrafficTestCase{
+			name: "without headers",
+			config: `
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: vs
+spec:
+  hosts:
+  - {{ .dstSvc }}
+  http:
+  - match:
+    - withoutHeaders:
+        end-user:
+          exact: jason
```

Comment:
can you add a test for withoutHeaders: regex: *

Reply:
will test it later

Reply:
Hello @hzxuzhonghu, added.